pkg/oc/util/tokencmd/negotiator_sspi.go

```diff
@@ -0,0 +1,188 @@
+// +build windows
+
+package tokencmd
+
+import (
+	"fmt"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/apimachinery/pkg/util/runtime"
+
+	"github.com/alexbrainman/sspi"
+	"github.com/alexbrainman/sspi/negotiate"
+	"github.com/golang/glog"
+)
+
+const (
+	// sane set of default flags, see sspiNegotiator.flags
+	// TODO make configurable?
+	flags = sspi.ISC_REQ_CONFIDENTIALITY |
+		sspi.ISC_REQ_INTEGRITY |
+		sspi.ISC_REQ_MUTUAL_AUTH |
+		sspi.ISC_REQ_REPLAY_DETECT |
+		sspi.ISC_REQ_SEQUENCE_DETECT
+
+	// separator used in fully qualified user name format
+	domainSeparator = `\`
```

Comment:
Might make more sense to have a regex definition here instead of using this variable and later a split.
With a comment that describe Windows Usernames here.